docker-compose.dev.yml

```diff
@@ -13,7 +13,7 @@ services:
       - ../optinist-docker-volumes/logs/:/app/logs
       - ../optinist-docker-volumes/studio_data/:/app/studio_data
     ports:
-      - "127.0.0.1:8000:8000"
+      - "0.0.0.0:8000:8000"
```

Comment:
I assume docker-compose.dev.yml is not used when using AWS ECS, but let me hear why you are updating this file.